scipy/interpolate/_interpolate.py

```diff
@@ -165,6 +165,8 @@ class interp2d:
         Spline interpolation based on FITPACK
     BivariateSpline : a more recent wrapper of the FITPACK routines
     interp1d : 1-D version of this function
+    RegularGridInterpolator : interpolation on a regular or rectilinear grid
+        in arbitrary dimensions (`interpn` wraps this class).
```

Comment:
Nit: Better add a separate heading for `interpn`

Reply:
Should be resolve by https://github.com/scipy/scipy/pull/16511/commits/52d52a16b083b5552fa7084ea0e02cd86e2b89a7